core/src/test/scala/unit/kafka/server/AlterReplicaLogDirsRequestTest.scala

```diff
@@ -37,6 +40,10 @@ class AlterReplicaLogDirsRequestTest extends BaseRequestTest {
 
   val topic = "topic"
 
+  override def brokerPropertyOverrides(properties: Properties): Unit = {
+    properties.put(ServerLogConfigs.LOG_INITIAL_TASK_DELAY_MS_CONFIG, "5000")
```

Comment:
Why do we need to wait 5 secs for it? I would say we can set to 0 to speed up the test. WDYT?

Reply:
Yeah, I also add `ServerLogConfigs.LOG_CLEANUP_INTERVAL_MS_CONFIG` as `1000`. The default value is 5 minutes. If we don't add it, after the initial task, we need to wait 5 minutes for the next one.